### ISSUE TITLE
Add support for Raspbian Buster (openssl 1.1.1c)

### ISSUE THREAD

**Issue by Millichrome** [jellyfin/jellyfin#1670]
**Describe the bug**
After installing Jellyfin on a Raspbian Buster based system, I was not able to access the webUI via browser. The service was reported as running but Jellyfin was not listening on port 8096 (other services were running just fine). Based on internet research this seems to be tied to the version of openssl that ships with Raspbian Buster (1.1.1c).

**To Reproduce**
1. Install Jellyfin as described in https://jellyfin.readthedocs.io/en/latest/administrator-docs/installing/#debian
2. Run "service jellyfin status" - the service is active
3. When you go to [Device IP]:8096, you get a generic "Unable to connect" FF error 
4. When you run "netstat -tnlp" Jellyfin is not listed, while other services (e.g. SSH server) are listed and work fine. 
5. Try to run Jellyfin manually "/usr/bin/jellyfin start"  and you'll get a "No usable version of the libssl was found" 

**Expected behavior**
Jellyfin webUI should be accessible. 

**System**
 - OS: Raspbian Buster
 - Browser: Firefox
 - Jellyfin Version: 10.3.7
 - Reverse proxy: None
 - OpenSSL 1.1.1c 28 May 2019

**Additional context**
The following article seems to describe this issue in more detail:

http://www.d0wn.com/installing-jellyfin-on-a-rasberry-pi4/ 

I did not try downgrading OpenSSL as described by the article.

**Comment by Wuerfelbecher** [comment]
I encountered this as i packaged Jellyfin for Fedora since on CentOS it build and on Fedora it would throw this error, that is why Jellyfin on Fedora requires the `compat-openssl10` package for the old OpenSSL1.0.X API (Luckily i commented why it is needed).

I tested this on Fedora 30 just to confirm it and it build Jellyfin without complaining, **with only OpenSSL 1.1.1** installed.

After some searching i found out that [dotnet already supports OpenSSL1.1.X](https://github.com/dotnet/corefx/issues/33870) (on dotnet 3.0) at least on Red Hat Distros tested on Fedora 30 and RHEL 8. [The pull request](https://github.com/dotnet/corefx/pull/34443) which back-ported the fix from dotnet 3.0 to 2.1/2.2 was already merged into dotnet quite some time ago.

I got no clue why this is not included in the Buster packages.
